android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/identitycredential/IdentityCredentialColors.kt

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.prompts.identitycredential
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+/**
+ * Represents the colors used by the IdentityCredential dialogs.
+ */
+data class IdentityCredentialColors(
+    val title: Color,
+    val description: Color,
+    val ripple: Color,
+)
+
```

Comment:
I was wondering if we could have a generic theme that could be used for other AC components that want to use compose, similar to what we have on [FirefoxColors](https://github.com/mozilla-mobile/firefox-android/blob/8af5f16e062c0871f362c41eb7b1a12c556a3c46/fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt#L233-L297), we don't need something that detailed but something generic enough that could be used in other components.